### PR TITLE
Feature branch

### PR DESCRIPTION
**Pull request by mikenath223**
This is a pull request to merge branch to master.

### CODE REVIEW

File: style/style.css [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+}
+
+.image-tile div:nth-child(3) {
+  background: url(/Backgrounds-and-Gradients/images/background4.jpg) center no-repeat;
+}
+
+.image-tile div:nth-child(4) {
+  background: url(/Backgrounds-and-Gradients/images/background5.jpg) center no-repeat;
+}
+
```
Comment (suretrust) [inline style/style.css]: Nice initiative as regards setting the background image from CSS.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+            <h1>What will your verse be?</h1>
+            <h2>Everyone has something to share. See how people are using</h2>
+            <h2>iPad in their unique ways to add to the world's story.</h2>
+            <h2 class="play">Watch the "Your Verse" film and explore the stories</h2>
+        </section>
+        <section class="image-tile">
+            <h1>.</h1>
+            <div></div>
+            <div></div>
+            <div></div>
```
Comment (suretrust) [inline index.html]: You can make your targeting more specific with classes or ids.

**Review by suretrust — CHANGES_REQUESTED** [review]
Hi @mikenath223,

**Great job thus far! :+1:**

I am not sure what went wrong but your web page looks far from ready. The problem could be from the way you targeted your image URL. Please refer to the image below:

screen

I cross-checked to ensure that I am on the right branch, I even went ahead to clone on my local machine to be sure. Kindly adjust appropriately and try to always check the live view of your code before submitting it for review.

**Please submit for another code review once this has been fixed.** _Happy coding! :tada:_

File: index.html (hunk 2 above)
Comment (mikenath223) [reply index.html]: Oh sorry i guess its a bug i will fix that now. Thanks

File: style/style.css [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+  display: flex;
+}
+
+.image-tile * {
+  width: 100%;
+  height: 220px;
+}
+
+.image-tile h1:nth-child(1) {
+  background: url(/Backgrounds-and-Gradients/images/background2.jpg) center no-repeat;
```
Comment (suretrust) [inline style/style.css]: The correct syntax here should be something like `background: url(../images/background2.jpg) center no-repeat;`. Kindly check and update throughout.

**Review by suretrust — CHANGES_REQUESTED** [review]

File: style/style.css (hunk 3 above)
Comment (mikenath223) [reply style/style.css]: Thanks alot i have done that.

**Review by Redvanisation — CHANGES_REQUESTED** [review]
# Changes required  🛑

### Hello @mikenath223   🌟

**Good work implementing the changes, but we still need to fix the issues below before moving on ⬇️** 

1. The navbar is too large, reduce the width please and give the elements a hover effect to be looking like the original page.

2. The text is not centered and it is bold, kindly center it and make it lighter, you can achieve that with something like: `font-weight: 300;`.

3. The background image is repeating, please fix that.
Screen Shot 2019-10-30 at 8 30 36 PM

4. Remove the big white margin between the 4 bottom elements.

### _After making the required changes please submit a new code review request_

### Good luck! 🍀

**Comment by mikenath223** [comment]
Alright cool i am working on implementing those changes now.
Thanks

**Comment by shshamim63** [comment]
### Status Approved!
___
Hi @mikenath223,
Well done :clap: , project contains all the parts that is needed to declare this project complete.
___
### Optional Changes
 Well there are still some spaces where you can improve to make this project as a perfect clone of the actual project. Check out the screenshots :point_down: ,
#### Actual one ->
number1

number2
___
#### Your's one ->
number3
number4
___
### This :+1:  from me, you can merge this project and don't forget to fill the project completion form.
___
### Happy Coding :clubs: 
#### Shakhawat Hossain
[Linkedin](https://www.linkedin.com/in/shakhawat-hossain-7750b0127/) | [Slack](Shakhawat Hossain)

**Comment by mikenath223** [comment]
Thanks alot i will implement optional changes and merge.
Cool.

**Comment by mikenath223** [comment]
All instructions for corrections have been followed merging now.
♥♥♥

**Pull request merged**